### PR TITLE
executor: tiny refactor, add sleep and cleanup for test case

### PR DESCRIPTION
**Pull request by cfzjywxk**
### What problem does this PR solve? 

Add cleanup for test case so that `ttlManager` will be closed

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

Release note

 - Write release note for bug-fix or new feature.

### CODE REVIEW

File: session/pessimistic_test.go [hunk 1]
```diff
@@ -748,10 +748,13 @@ func (s *testPessimisticSuite) TestInnodbLockWaitTimeoutWaitStart(c *C) {
 		}()
 		_, err = tk2.Exec("select * from tk where c1 = 1 for update")
 	}()
+	time.Sleep(time.Millisecond * 30)
```
Comment (coocood) [inline session/pessimistic_test.go]: Is this necessary?

Comment (cfzjywxk) [reply session/pessimistic_test.go]: There may be sometime before the tk2 statement reach `failpoint` code in `2pc`

**Comment by coocood** [comment]
LGTM

**Pull request merged**